blog/templates/blog/post_list.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<html>
+<head>
+    <title>{% trans "Django Girls blog" %}</title>
+</head>
+<body>
+<div>
+    <h1><a href="/">{% trans "Django Girls Blog" %}</a></h1>
+</div>
+
+<div>
+    <p>{% trans "published: 14.06.2014, 12:14" %}</p>
```

Comment:
Date part is variable, so it shouldn't affect static localizable part. Well, there's actually a way of having i18n in code parts, but we'll get to it eventually :)